minpy/nn/solver.py

```diff
@@ -4,35 +4,39 @@
 from __future__ import print_function
 import time
 
+import minpy
+
 from minpy.nn import optim, init
 from minpy import core
 import minpy.numpy as np
 
+from minpy.visualize.writer import LegacySummaryWriter as SummaryWriter
+import minpy.visualize.summaryOps as summaryOps
 
 class Solver(object):
     """
     A Solver encapsulates all the logic necessary for training classification
     models. The Solver performs stochastic gradient descent using different
     update rules defined in optim.py.
-
+    
```

Comment:
Fix these trailing whitespaces. You could use `python ./dmlc-core/scripts/lint.py minpy python <your code>` to check for codestyle problems.